docs/ios/xcsync.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Xcode Sync (xcsync) for .NET MAUI
+description: [Xcsync is designed to provide .NET developers with improved support for editing Apple specific files]
+author: haritha-mohan, mcumming
+ms.date: [08/05/2024]
+---
+# Xcode Sync (Xcsync)
+
+**This article applies to:** ✔️ .NET Core 9 Preview 7 SDK and later versions
```

Comment:
You can remove this line, instead add the following to the metadata:

`monikerRange: ">=net-maui-9.0"`